Formula/mplayer.rb

```diff
@@ -3,6 +3,7 @@ class Mplayer < Formula
   homepage "https://mplayerhq.hu/"
   url "https://mplayerhq.hu/MPlayer/releases/MPlayer-1.4.tar.xz"
   sha256 "82596ed558478d28248c7bc3828eb09e6948c099bbd76bb7ee745a0e3275b548"
+  license all_of: ["GPL-2.0-only", "GPL-2.0-or-later"]
```

Comment:
How can it be both only GPL2 but also later?

Reply:
From `Copyright`:
```
Most files in MPlayer are licensed under the GNU General Public License,
version 2 with an explicit "or later" clause or a compatible license. An
exception is the file libmpdemux/demux_ty_osd.c, which is version 2 only.
Thus MPlayer as a whole can be redistributed under the terms of the GNU
General Public License, version 2, see the LICENSE file for details.
```
Just reporting what's in the headers

Reply:
Frol what it says in the header it should actually be one or the other. I don't know what we're doing of the mentioned file, though. We may indeed have to select the -only option.